apps/meteor/client/hooks/useAppSlashCommands.spec.tsx

```diff
@@ -33,9 +34,18 @@ const mockApiResponse = {
 
 describe('useAppSlashCommands', () => {
 	let mockGetSlashCommands: jest.Mock;
+	let queryClient: QueryClient;
 
 	beforeEach(() => {
 		mockGetSlashCommands = jest.fn().mockResolvedValue(mockApiResponse);
+		queryClient = new QueryClient({
+			defaultOptions: {
+				queries: { retry: false },
+				mutations: { retry: false },
+			},
+		});
+
+		jest.spyOn(queryClient, 'invalidateQueries');
```

Comment:
just to make sure, you added the support to inject queryClient in the mock provider to be able to call this invalidateQueries, right?